src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -12,19 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets.utils;
+package com.google.sps.servlets;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class Statistics {
   // Aggregated score for a list of comments  
-  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
-  private List<Double> aggregateValues;
+  // Contains number of sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private List<Double> sentimentScores;
+  private Map<Range, Integer> aggregateValues;
```

Comment:
I'm not sure we want the `sentimentScores` to be apart of the Statistics object since then they'll be included when we serialize this object to JSON.